drivers/sensor/mhz19b/mhz19b.c

```diff
@@ -240,11 +240,11 @@ static int mhz19b_attr_get(const struct device *dev, enum sensor_channel chan,
 
 static int mhz19b_sample_fetch(const struct device *dev, enum sensor_channel chan)
 {
-	if (chan != SENSOR_CHAN_CO2) {
+	if (chan == SENSOR_CHAN_CO2 || chan == SENSOR_CHAN_ALL) {
+		return mhz19b_poll_data(dev, MHZ19B_CMD_IDX_GET_CO2);
+	} else {
```

Comment:
Else is not required

```suggestion
static int mhz19b_sample_fetch(const struct device *dev, enum sensor_channel chan)
{
	if (chan == SENSOR_CHAN_CO2 || chan == SENSOR_CHAN_ALL) {
		return mhz19b_poll_data(dev, MHZ19B_CMD_IDX_GET_CO2);
	}

	return -ENOTSUP;
}
```

Reply:
Thanks, I updated my commit, removed `else`.